specification/Overview.md

```diff
@@ -33,7 +68,12 @@ organised by data category (saveframe).
     deposition must preserve the original names (possibly using a
     deposition-specific namespace for the new names).
 
-    4. For the common standard residues (20 amino acids, 4 DNA and 4 RNA
+    4. Structures must be given in mmCIf format. In this format the author naming
```

Comment:
This point should be highlighted for the software developers. Their example NEF file should be associated with a mmCIF file with proper NEF atom mapping using author tags.